### DISCUSSION TITLE
PTT time

### DISCUSSION THREAD

**Discussion by uguraltinsoy** [markqvist/Reticulum#611] [General]
I am using the LibAPRS library and my problem is that the default PTT time is too short and the signal is not fully transmitted. How can I bypass it and make my own continuous TX, the signal is much cleaner.

https://www.youtube.com/watch?v=PrCiLKZh58M

**Comment by markqvist** [comment]
This is not really relevant to Reticulum, and LibAPRS is not a package I am actively working on anymore.

That being said, Baofeng and similar cheap radios are quite bad for digital stuff. They have very slow RX/TX turnaround time, they are slow to open the squelch on received signal, and slow to open the transmitter once PTT has been triggered.

With such radios, you need to run them in open squelch for the receive, and you need to increase the preamble time in the modem to give the radio more time to open up. Such radios can often need a preamble time of 250ms or more, where a faster radio would only need 40ms or so to open. They also need to have a pretty carefully adjusted receive audio level to decode properly.

It's not the best hardware to use for digital, unfortunately.
